sdcm/cluster.py

```diff
@@ -2151,7 +2151,7 @@ def install_mgmt(self, scylla_mgmt_repo, auth_token, package_url=None):
             self.remoter.run('sudo systemctl restart scylla-manager.service')
             res = self.remoter.run('sudo systemctl status scylla-manager.service')
 
-        manager_prometheus_port = self.parent_cluster.params.get("manager_prometheus_port", 5090)
+        manager_prometheus_porttest = self.parent_cluster.params.get("manager_prometheus_port", 5090)
```

Comment:
pre-commit is failing cause of this change...

Reply:
Fixed, thanks!